tests/test_dataset_common.py

```diff
@@ -231,6 +232,13 @@ def test_builder_configs(self, dataset_name):
         if builder_configs[0] is not None:
             all(self.assertTrue(isinstance(config, BuilderConfig)) for config in builder_configs)
 
+    def test_task_implementations(self, dataset_name):
+        if dataset_name != "rotten_tomatoes":
+            return
+
+        # this is failing at the moment since the new code for "rotten_tomatoes" is not uploaded.
+        load_dataset(dataset_name, as_task=tasks.ClassificationSingleLabelDataset.id)
```

Comment:
it would be good to have a more general test than check for the dataset names.
Maybe by checking the available tasks for the builder instance ?